README.md

```diff
@@ -17,15 +17,28 @@ Lint JS, JSX, TypeScript, and more inside Markdown.
 
 ### Installing
 
-Install the plugin alongside ESLint v6 or greater:
+Install the plugin alongside ESLint v8 or greater:
 
 ```sh
 npm install --save-dev eslint eslint-plugin-markdown
 ```
 
 ### Configuring
 
-Extending the `plugin:markdown/recommended` config will enable the Markdown processor on all `.md` files:
+In your `eslint.config.js` file, import `eslint-plugin-markdown` and included the recommended config to enable the Markdown processor on all `.md` files:
+
+```js
+// eslint.config.js
+import markdown from "eslint-plugin-markdown";
+
+export default [
+    ...markdown.configs.recommended
+
+    // your other configs here
+];
+```
+
+If you are still using the deprecated `.eslintrc.js` file format for ESLint, you can extend the `plugin:markdown/recommended` config to enable the Markdown processor on all `.md` files:
```

Comment:
```suggestion
If you are still using the deprecated `.eslintrc.js` file format for ESLint, you can extend the `plugin:markdown/recommended-legacy` config to enable the Markdown processor on all `.md` files:
```

Reply:
The same for example on line 46: it should be ` extends: "plugin:markdown/recommended-legacy"`